pybossa/view/account.py

```diff
@@ -1156,30 +1156,37 @@ def add_metadata(name):
     return redirect(url_for('account.profile', name=name))
 
 
-def bookmarks_dict_to_array(bookmarks_dict):
+def bookmarks_dict_to_array(bookmarks_dict, order_by, desc):
+    order_by = "name" if order_by is None else order_by
+    desc = False if desc is None else desc
```

Comment:
Do the following also meet your need?
```
order_by =  order_by or "name"
desc = desc or False
```
There is a slightly different from your original logic (when order_by or desc is empty, but looks to me fits better)

Reply:
refactored via https://github.com/bloomberg/pybossa/pull/843/commits/81b103681f39a9575aeedfcb5778663d84b2f9df